Assets/Scripts/Robot/MonaRobot.cs

```diff
@@ -51,6 +51,8 @@ public class MonaRobot : MonoBehaviour, ISimulationUnit, IPointerEnterHandler, I
 
         public OnRobotSelectedDelegate OnRobotSelected = _ => { };
 
+        public int AssignedPartition { get; set; }
```

Comment:
Should anybody be able to set it?

Reply:
Maybe we should set it in the constructor instead?

Reply:
I also really dont like it being here it seems limiting

Reply:
Will maybe be moved in another PR.